tests/src/traffic/steal.rs

```diff
@@ -567,7 +567,7 @@ mod steal {
 
         let addr = SocketAddr::new(host.trim().parse().unwrap(), port as u16);
         let mut stream = TcpStream::connect(addr).await.unwrap();
-        stream.write(tcp_data.as_bytes()).unwrap();
+        stream.write(tcp_data.as_bytes()).await.unwrap();
```

Comment:
```suggestion
        stream.write_all(tcp_data.as_bytes()).await.unwrap();
```

Like @Razz4780 and Clippy suggested.